Add clarity to WPMS documentation about requesting multisites (#9332)
pantheon-systems/documentation: source/content/guides/multisite/01-introduction.md

## Request a WordPress Multisite
WordPress Multisite requires a special configuration that is only available to select customers. Access to WordPress Multisite requires a Gold, Platinum, or Diamond level [Workspace plan](https://pantheon.io/plans/pricing).

> By default, Pantheon employees must create new Multisites on your behalf. This is because they use the `wordpress_network` [framework](/glossary/#framework) which includes Multisite specific Nginx configurations and differs from the default `wordpress` framework. Reach out to your account manager to request site creation for a new WordPress Multisite.

Author: ```suggestion
By default, Pantheon employees must create a new Multisite upstream on your behalf. This is because they use the `wordpress_network` [framework](/glossary/#framework) which includes Multisite specific Nginx configurations and differs from the default `wordpress` framework. Reach out to your account manager to request site creation for a new WordPress Multisite.
```

> By default, Pantheon employees must create the WordPress Network upstream on your behalf so your team can create new Multisites on the platform. This is because they use the `wordpress_network` [framework](/glossary/#framework) which includes Multisite specific Nginx configurations and differs from the default `wordpress` framework. Reach out to your account manager to request site creation for a new WordPress Multisite.

Author: ```suggestion
By default, Pantheon employees must create the WordPress Multisite upstream on your behalf so your team can create new Multisites on the platform. This is because they use the `wordpress_network` [framework](/glossary/#framework) which includes Multisite specific Nginx configurations and differs from the default `wordpress` framework. Reach out to your account manager to request site creation for a new WordPress Multisite.
```

> By default, Pantheon employees must create the WordPress Multisite upstream on your behalf so your team can create new Multisites on the platform. This is because they use the `wordpress_network` [framework](/glossary/#framework) which includes Multisite specific Nginx configurations and differs from the default `wordpress` framework. Reach out to your account manager to request site creation for a new WordPress Multisite.

Reviewer: ```suggestion
Pantheon employees must enable a WordPress Multisite custom upstream on your behalf so your team can create new Multisites self-serve in your professional workspace. This is because WordPress Multisites use the `wordpress_network` [framework](/glossary/#framework), which includes Multisite specific Nginx configurations and differs from the default `wordpress` framework. Reach out to your account manager to get started, or if you don't have an account manager, you can [contact sales](https://pantheon.io/contact-us).
```

> If you want to create new WordPress Multisites self-serve and on-demand in the Pantheon Dashboard, reach out to your account manager and request a [Custom Upstream](/guides/custom-upstream) for your workspace based on Pantheon's WordPress Multisite framework.

Reviewer: ```suggestion
```

Given the edits to the prior paragraph, this line is no longer needed

Existing WordPress sites cannot be converted to WordPress Multisites (since they are different frameworks), however you can spin up a new WordPress Multisite and then [manually migrate](/migrate-manual) the code, database and files from your existing site. 

> If you don't have an account manager, you can [contact sales](https://pantheon.io/contact-us).

Reviewer: ```suggestion
```

This feels out of place, relocated it to the end of line 44 in my suggestion above

> Setting up an existing WordPress site  on Pantheon as a WordPress multisite without the correct framework can result in unexpected errors due to the improper Nginx configuration. If you have an existing WordPress site that you want to make into a multisite but the upstream does not use the `wordpress_network` framework (visible in the Pantheon Dashboard), and a WordPress Multisite upstream has been created for you, you can create a new site based on that upstream and push your codebase pushed to the new site repository (since WordPress Multisite is a core component of WordPress and uses the same code).

Reviewer: ```suggestion
Existing WordPress sites cannot be converted to WordPress Multisites (since they are different frameworks), however you can spin up a new WordPress Multisite and then [manually migrate](/migrate-manual) the code, database and files from your existing site. 
```

WordPress Multisite requires a special configuration that is only available to select customers. Access to WordPress Multisite requires a Gold, Platinum, or Diamond level [Workspace plan](https://pantheon.io/plans/pricing).

By default, Pantheon employees must create new Multisites on your behalf. This is because they use a different platform framework (which includes the appropriate Nginx configuration) than the default `wordpress` framework typically available to customers. Reach out to your account manager to request site creation for a new WordPress Multisite.

If you want to create new WordPress Multisites self-serve and on-demand in the Pantheon Dashboard, reach out to your account manager and request a [Custom Upstream](/guides/custom-upstream) for your workspace based on Pantheon's WordPress Multisite framework.

Existing WordPress sites on Pantheon cannot be converted to a multisite due to the framework limitation. Instead, if a WordPress Multisite upstream has been created for you, a new site can be created based on that upstream and the codebase pushed to that site repository (since WordPress Multisite is a core component of WordPress and uses the same code).

> **Note:** Any non-multisite WordPress site can be [configured](/guides/multisite/config) as a WordPress Multisite. However, it is not recommended to set up your WordPress site to be a Multisite if your upstream is not using the correct framework.

Reviewer 2: Can you rewrite this sentence to be more like "If you try to changing a WordPress instance on Pantheon to Multisite by only making changes within WordPress without also making this "platform framework" change (that updates nginx and other settings) you will get errors."
